### PR TITLE
🐛 use false dual for tutorial1 linearSVC

### PR DESCRIPTION
**Pull request by sammaxwellxyz**
Hi,

Without dual=False, the algorithm raises
```py
/usr/local/lib/python3.7/site-packages/sklearn/svm/_base.py:977: ConvergenceWarning: Liblinear failed to converge, increase the number of iterations.
  "the number of iterations.", ConvergenceWarning)
```

https://scikit-learn.org/stable/modules/generated/sklearn.svm.LinearSVC.html,
suggests preferring false when n_samples > n_features.

Having changed and re-run the cell, no warning is raised, and the accuracy seems to be increased.

Apologies if this it not actually desired at all :)

### CODE REVIEW

**Comment by rlenain** [comment]
Hi there,
Thanks for your suggestion!
This looks interesting. However, I believe that we want to keep this notebook the most simple possible (the goal not being to get the best accuracy). 
Thanks though!
